src/Calculator.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Acme;
+
+class Calculator
+{
+    public $displayyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyaaaayyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyy;
+
+    function div(int $a, int $b): int
+    {
+        return $a / $b;
+    }
+
+    var $aaa;
+
+    var $bbb;
+
+    public function sayHello(DateTimeImutable $date): void
```

Comment:
**[phpstan]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Parameter $date of method Acme\Calculator::sayHello() has invalid typehint type Acme\DateTimeImutable.